Spanish Translation V2.0

alexsf93 opened the pull request:
Accents have been removed

alexsf93 committed:
Spanish Translation V2.0

Accents have been removed

alexsf93 committed:
Spanish Translation v2.0

email_change.lang
empire.lang
firstlogin.lang

alexsf93 committed:
Spanish Translation v2.0

Spanish translation of:
fleet.lang
FleetMission_MissileAttack.lang
tasks_infobox.lang
texts.lang
topmenu.lang

alexsf93 committed:
Spanish Translation V.2.0 OF:

fleetshortchut.lang
generate_sig.lang
jumpgate.lang
login.lang
logout.lang
stat.lang

mdziekon commented on language/es/alliance.lang:
```diff
@@ -0,0 +1,575 @@
+
+// Inviting other Users
+
+$_Lang['Ally_INV_Title']        = 'Alinza - Envie una invitacion';
```
```suggestion
$_Lang['Ally_INV_Title']        = 'Alianza - Envie una invitacion';
```

mdziekon commented on language/es/alliance.lang:
```diff
@@ -0,0 +1,575 @@
+$_Lang['ADM_HandOverTitle']      = 'Alianza - Entrega';
+
+$_Lang['ADM_OnlyOwnerInAlly']    = 'No puedes ceder esta alianza, porque eres su unico miembro!';
+$_Lang['ADM_SureHandOver']       = '¿Estas seguro de que quieres ceder esta alianza? \ NCuando la entregues, tu rango se cambiara al rango de tu sucesor.!';
```
```suggestion
$_Lang['ADM_SureHandOver']       = '¿Estas seguro de que quieres ceder esta alianza? \n Cuando la entregues, tu rango se cambiara al rango de tu sucesor.!';
```

mdziekon commented on language/es/fleet.lang:
```diff
@@ -0,0 +1,278 @@
+$_Lang['fl2_SendButton']                         = 'Enviar flota';
+
+$_Lang['fl2_confirm_allypact_attack']            = '¿Estas seguro de que quieres atacar a un jugador que esta en una alianza que tiene un pacto con la alianza en la que estas actualmente?';
+$_Lang['fl2_testacctarget']                      = '¡Advertencia! \ N¡Las coordenadas del objetivo pertenecen a una de las cuentas de prueba de la Administracion del juego! \n Cualquier accion que involucre a este objetivo sin el permiso del administrador podria resultar en una prohibicion!';
```
```suggestion
$_Lang['fl2_testacctarget']                      = '¡Advertencia! \n ¡Las coordenadas del objetivo pertenecen a una de las cuentas de prueba de la Administracion del juego! \n Cualquier accion que involucre a este objetivo sin el permiso del administrador podria resultar en una prohibicion!';
```

mdziekon reviewed (changes requested):
Overall looks great, good work! :+1: 
Just a couple of minor fixes and this Pull Request is ready to be merged.

(As you can see, this took me quite some time to review, so in the follow-up PRs I would recommend keeping them a bit shorter than this. You could also split next translations into separate PRs, as there's usually nothing preventing you from submitting multiple PRs in parallel).

alexsf93 committed:
Spanish translation v.2.0

admin.lang
\admin\addMoon.lang
\admin\index.php
lostcode.lang
lostpassword.lang
moon_sim.lang
rules.lang
search.lang

mdziekon reviewed (changes requested):
New changes look fine, nothing to fix there.
The old comments still stand, so please fix them before we move further.
